cEP-0005.md

```diff
@@ -124,6 +124,32 @@ class RedundancyBear(LocalBear):
             yield Result.from_values(aspect=aspect, ...)
 ```
 
+## Naming of aspects and tastes
+
+## aspects
+
+An aspect should just talk about a property of something, without qualifying
+it. It should be named something like `Length` rather than `TooLong`. Then it
+should be given the tastes.
+For e.g. an aspect can be named as `Metadata.CommitMessage.Shortlog.Length` to
+represent the length property of the shortlog (first line) of a commit message.
+
+## tastes
+
+tastes are the values that are used to initialize the aspect settings. It
+should not be something like `allow` or `check` which enables or disables an
+aspect, rather it should be something more like a value which describes an
+aspect in a specific, measurable way. It can be given as much context as
+needed but not more.
+For e.g. for the aspect `Metadata.CommitMessage.Shortlog.`
+`FirstCharacter`, set the taste `shortlog_starts_upper_case` as true if the
+shortlog must begin with a upper case letter consistently, and false if the
+opposite is true.
+
+__Note:__ aspects and tastes are orthogonal concepts and should never overlap.
+Therefore extreme caution must be taken while naming aspects and tastes to
+avoid any discrepancy and inconsistency in the future.
+
 The concept of aspects allows us to implement a consistency check and reduce
```

Comment:
A new section starts here, not related to naming anymore. So we need some `##` heading at this point